### PR TITLE
Alsaaidi

### PR DESCRIPTION
**Pull request by loushiano**
I am waiting for you reviews guys

### CODE REVIEW

File: src/main/java/com/carleton/CapstoneSystem/models/Program.java [hunk 1]
```diff
@@ -1,7 +1,18 @@
 package com.carleton.CapstoneSystem.models;
 
 public enum Program {
-    SOFTWARE_ENGINEERING,COMPUTER_ENGINEERING,BIOMEDICAL_ENGINEERNG;
+    SOFTWARE_ENGINEERING("SE"),COMPUTER_ENGINEERING("CE"),BIOMEDICAL_ENGINEERNG("BE");
```
Comment (mohameddahrouj) [inline src/main/java/com/carleton/CapstoneSystem/models/Program.java]: Are there any other programs we should add?

Comment (tosinoni) [reply src/main/java/com/carleton/CapstoneSystem/models/Program.java]: I agree, let's not be lazy. Please kindly add all the programs in this PR.

File: src/test/java/com/carleton/CapstoneSystem/models/ProfessorTest.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.carleton.CapstoneSystem.models;
+
+import org.junit.Test;
+
+import javax.persistence.Entity;
+import javax.persistence.Inheritance;
+
+public class ProfessorTest {
+    @Test
```
Comment (tosinoni) [inline src/test/java/com/carleton/CapstoneSystem/models/ProfessorTest.java]: we should be testing all the getters and setters too technically. That's why the code coverage is complaining.

File: src/main/java/com/carleton/CapstoneSystem/models/Program.java (hunk 1 above)
Comment (alialsaaidi) [reply src/main/java/com/carleton/CapstoneSystem/models/Program.java]: I will add them can you please provide a list of what they are?

File: src/test/java/com/carleton/CapstoneSystem/models/ProfessorTest.java (hunk 2 above)
Comment (alialsaaidi) [reply src/test/java/com/carleton/CapstoneSystem/models/ProfessorTest.java]: ok I will make the right tests for those

File: src/main/java/com/carleton/CapstoneSystem/models/Program.java (hunk 1 above)
Comment (loushiano) [reply src/main/java/com/carleton/CapstoneSystem/models/Program.java]: you can check the programs we have from the last deploy.

File: src/test/java/com/carleton/CapstoneSystem/models/ProfessorTest.java (hunk 2 above)
Comment (loushiano) [reply src/test/java/com/carleton/CapstoneSystem/models/ProfessorTest.java]: same comment as tosin

File: src/main/java/com/carleton/CapstoneSystem/models/Program.java (hunk 1 above)
Comment (mohameddahrouj) [reply src/main/java/com/carleton/CapstoneSystem/models/Program.java]: image

**Comment by mohameddahrouj** [comment]
image
 We got the names of the programs from here.

**Pull request merged**